SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1356,11 +1351,6 @@ AActor* USpatialReceiver::FindSingletonActor(UClass* SingletonClass)
 	{
 		return FoundActors[0];
 	}
-	else
```

Comment:
This message was firing in scenarios that were perfectly normal under snapshot reload or server restart